<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HV-KB390L Configuration</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageAbout">
    <attribute name="title">
     <string>Abo&amp;ut</string>
    </attribute>
    <layout class="QVBoxLayout" name="layoutAbout">
     <item>
      <widget class="QLabel" name="labelLogo">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="pixmap">
        <pixmap resource="../res/hv-kb390l-config.qrc">:/app/logo</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerAbout">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelText">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1&gt;HV-KB390L&lt;/h1&gt;&lt;p&gt;Configuration utility&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Version %1&lt;/span&gt;&lt;/p&gt;&lt;p&gt;build %2&lt;br/&gt;&lt;/p&gt;&lt;p&gt;This program &lt;span style=&quot; font-weight:600;&quot;&gt;is not &lt;/span&gt;an official utility from the &lt;a href=&quot;https://www.prohavit.com/products/hv-kb390l-low-profile-mechanical-keyboard/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;product vendor&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;It is strongly recommended to use the software from the official website:&lt;br/&gt;&lt;a href=&quot;http://www.havit.hk/downloads/&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline; color:#0000ff;&quot;&gt;http://www.havit.hk/downloads/&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;Use this utility only if your system is not supported by the manufacturer,&lt;br/&gt;or if you need some extra features that the official software does not provide.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Absolutely no warranty&lt;/span&gt;. Perhaps (and in some cases definitely), the device will be broken.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;Copyright (c) 2018 &lt;a href=&quot;https://github.com/pbludov&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Pavel Bludov&lt;/span&gt;&lt;/a&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="pageButtons1">
    <attribute name="title">
     <string>Row &amp;1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageButtons2">
    <attribute name="title">
     <string>Row &amp;2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageButtons3">
    <attribute name="title">
     <string>Row &amp;3</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageButtons4">
    <attribute name="title">
     <string>Row &amp;4</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageButtons5">
    <attribute name="title">
     <string>Row &amp;5</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageButtons6">
    <attribute name="title">
     <string>Row &amp;6</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageMacros">
    <attribute name="title">
     <string>&amp;Macros</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageSpeed">
    <attribute name="title">
     <string>&amp;Speed</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="pageLight">
    <attribute name="title">
     <string>&amp;Light</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/hv-kb390l-config.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onPreparePage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
